src/components/Carousel.js

```diff
@@ -602,6 +649,10 @@ class Carousel extends Component {
             }
         }
 
+        const itemsClone = [...this.renderItems(true)];
```

Comment:
Don't need to spread the items here if you are not reusing it. Just `const itemsClone = this.renderItems(true);` is enough.